src/reducers/restaurantReducer.js

```diff
@@ -3,21 +3,30 @@ import {
     FETCH_RESTAURANTS_STARTED,
     FETCH_RESTAURANTS_SUCCESS,
     ADD_RESTAURANT_FAILURE,
-    ADD_RESTAURANT_SUCCESS,
+    ADD_RESTAURANT_SUCCESS
 } from '../actions/types';
 
-export default (state = { data: [] }, action) => {
+const INITIAL_STATE = {
+    data: [],
+    filters: {
+        max_delivery_time: '',
+        rating: '',
```

Comment:
if `rating` is a number, i'd stick with better conventions such as initializing with 0 or null instead of a different non-related type such as a string.